Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM jimmycuadra/rust:1.16.0
+FROM debian:jessie-slim
```

Comment:
are we able to use alpine? don't have a strong preference, but if we're going for small, alpine should win every time